packages/snaps-jest/src/types.ts

```diff
@@ -253,16 +266,26 @@ export type Snap = {
    * `endowment:cronjob` permission.
    * @returns The response promise, with extra {@link SnapRequestObject} fields.
    */
-  runCronjob(cronjob: CronjobOptions): SnapRequest;
+  onCronjob(cronjob?: Partial<CronjobOptions>): SnapRequest;
 
   /**
-   * Makes a request to the snap to render the snaps home page.
+   * Run a cronjob in the snap. This is similar to {@link request}, but the
+   * request will be sent to the `onCronjob` method of the snap.
    *
-   * This method takes no arguments.
+   * @param cronjob - The cronjob request. This is similar to a JSON-RPC
+   * request, and is normally specified in the snap manifest, under the
+   * `endowment:cronjob` permission.
+   * @returns The response promise, with extra {@link SnapRequestObject} fields.
+   * @deprecated Use {@link onCronjob} instead.
```

Comment:
Should we just break these in the next release instead of keeping them as deprecated? Since we have breaking changes regardless

Reply:
The main breaking change right now is the removal of network mocking. If you don't use network mocking, everything should still work. I think it's better if we give developers some time to update the deprecated functions, and make it easier to update to the new version.

Reply:
Fair enough 👍